### PR TITLE
Lifecourse view layout

### PR DESCRIPTION
**Pull request by Arth101**

### CODE REVIEW

File: src/assets/styling/utility.scss [hunk 1]
```diff
@@ -19,8 +25,20 @@
...
   margin: 0 !important;
 }
+
+.u-ml-2 {
+  margin-left: 10px !important;
+}
+
+.u-mr-2 {
+  margin-right: 10px !important;
+}
```
Comment (hypesystem) [inline src/assets/styling/utility.scss:44]: Missing newline @ eof

File: src/assets/styling/utility.scss [hunk 2]
```diff
@@ -19,8 +25,20 @@
...
 .u-margin-0 {
   margin: 0 !important;
 }
+
+.u-ml-2 {
+  margin-left: 10px !important;
+}
+
+.u-mr-2 {
+  margin-right: 10px !important;
```
Comment (hypesystem) [inline src/assets/styling/utility.scss:43]: I think we should define these using rem? `0.625rem` in this case.

File: src/assets/styling/lifecourse-page.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+.lls-lifecourse-footer__entry {
+  font-size: 1rem;
+  font-weight: 400;
+}
+.lls-lifecourse-footer__entry-label {
+  color: $grey--light;
+  font-size: 0.875rem;
+  font-weight: 400;
+}
```
Comment (hypesystem) [inline src/assets/styling/lifecourse-page.scss:20]: Missing newline @ eof

File: src/assets/styling/lifecourse-page.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
+.lls-lifecourse-header__text {
+  display: flex;
+}
+
+.lls-lifecourse-header__links {
+
```
Comment (hypesystem) [inline src/assets/styling/lifecourse-page.scss:6]: odd newline here i think?

File: src/app/search-result-list/search-result-list.component.html [hunk 5]
```diff
@@ -108,32 +108,26 @@
...
-                </div>
-            </div>
+                    <li class="page-item" [ngClass]="{ disabled: pagination.current >= pagination.last }">
+                        <a class="page-link" [tabindex]="pagination.current >= pagination.last ? -1 : 0"  [routerLink]="['./', { 'size': pagination.size, 'page': pagination.last } ]" [queryParams]="queryParams">Næste</a>
+                    </li>
+                </ul>
+            </nav>
         </div>
     </div>
 </div>
```
Comment (hypesystem) [inline src/app/search-result-list/search-result-list.component.html:133]: missing newline @ eof

File: src/app/life-course/life-course.component.html [hunk 6]
```diff
@@ -1,30 +1,41 @@
...
-    </div>
-
-    <ul class="row nav nav-pills">
-        <li class="nav-item active">Livsforløb</li>
-        <li class="nav-item">Familierelationer</li>
-    </ul>
-
-    <app-person-appearance-item [item]="pa" class="row border p-2 mb-2" *ngFor="let pa of pas"></app-person-appearance-item>
+    </section>
 </div>
```
Comment (hypesystem) [inline src/app/life-course/life-course.component.html:41]: Missing newline @ eof

File: src/app/life-course/life-course.component.html [hunk 7]
```diff
@@ -1,30 +1,41 @@
...
             <div>
-                <strong>Fødested:</strong> {{ pas[0].birth_place }}
+                <a class="lls-btn u-mr-2" href="">Download</a>
+                <a class="lls-btn" href="">Download</a>
             </div>
-
-            <div>
-                <strong>Fødselsår:</strong> n/a
+        </div>
+    </section>
```
Comment (hypesystem) [inline src/app/life-course/life-course.component.html:20]: I think for this entire header you should have extracted and reused the one from the source view?

image

File: src/app/life-course/life-course.component.html [hunk 8]
```diff
@@ -1,30 +1,41 @@
...
+    <app-person-appearance-item [item]="pa" class="mx-2" *ngFor="let pa of pas"></app-person-appearance-item>
+    <section class="lls-lifecourse-footer">
+        <div class="lls-lifecourse-footer__entry">
+            <div class="lls-lifecourse-footer__entry-label">
+                Forløbs ID
             </div>
+            <p>{{ lifecourseId }}</p>
```
Comment (hypesystem) [inline src/app/life-course/life-course.component.html:27]: The first part of the ID here could have been based on the one in the source view, too, including the href :smile:

File: src/app/life-course/life-course.component.html (hunk 7 above)
Comment (hypesystem) [reply src/app/life-course/life-course.component.html:20]: Maybe not extracting the content, but at least the styling?

File: src/app/life-course/life-course.component.html [hunk 9]
```diff
@@ -1,30 +1,41 @@
...
+                <p>~{{ latestPersonAppearance.source_year - latestPersonAppearance.age_clean }}</p>
             </div>
-
+        </div>
+        <div class="lls-lifecourse-header__links lls-columns-12 lls-columns-4--md">
+            <a href="">Min søgehistorik</a>
             <div>
-                <strong>Fødested:</strong> {{ pas[0].birth_place }}
+                <a class="lls-btn u-mr-2" href="">Download</a>
+                <a class="lls-btn" href="">Download</a>
```
Comment (hypesystem) [inline src/app/life-course/life-course.component.html:17]: Signe told us at the start that we shouldn't have the download buttons in this iteration

**Review by hypesystem — CHANGES_REQUESTED** [review]
Reviewed :smile:

Generally the code looks really nice, but I think we should align the source view and life course view headers and footers, as they are designed identically in Sketch :smile:

Måske værd at reviewe og merge #22 først, fordi den opdaterer footeren.

File: src/app/life-course/life-course.component.html (hunk 9 above)
Comment (Arth101) [reply src/app/life-course/life-course.component.html]: I'll remove them :)

File: src/app/life-course/life-course.component.html (hunk 7 above)
Comment (Arth101) [reply src/app/life-course/life-course.component.html:21]: Makes sense. It is extracted to `data-page-header`.

File: src/app/life-course/life-course.component.html (hunk 8 above)
Comment (Arth101) [reply src/app/life-course/life-course.component.html:28]: This ID seems to be just "1" all the time. I've made a TODO comment but I think we should fix it later and get this merged.

**Review by hypesystem — APPROVED** [review]

**Pull request merged**